nucliadb_utils/nucliadb_utils/tests/conftest.py

```diff
@@ -25,3 +30,23 @@
     "nucliadb_utils.tests.local",
     "nucliadb_utils.tests.clandestined",
 ]
+
+
+@pytest.fixture(scope="function")
```

Comment:
Should we move it to its own file? Like `nucliadb_utils.tests.pg` So that we can import it on other components's tests

Reply:
I was actually thinking the opposite and that it should be in it's own test file.

Do you think we'll want to ever use it in other tests?

I think we have too many shared fixtures.